src/dmd/target.d

```diff
@@ -878,6 +878,22 @@ extern (C++) struct Target
         }
     }
 
+    /**
+     * Return `true` if the callee invokes destructors for arguments.
+     */
+    extern (C++) bool isCalleeDestroyingArgs(TypeFunction tf)
+    {
+        // On windows, the callee destroys arguments always regardless of function linkage,
+        // and regardless of whether the caller or callee cleans the stack.
+        if (global.params.isWindows)
+            return true;
+
+        const bool calleeCleansStack = tf.linkage == LINK.pascal || tf.linkage == LINK.windows;
```

Comment:
`LINK.pascal` has been removed in the interim.

Reply:
ok